src/wp-includes/rest-api/endpoints/class-wp-rest-global-styles-controller.php

```diff
@@ -563,7 +563,8 @@ public function get_theme_item( $request ) {
 		}
 
 		if ( rest_is_field_included( 'styles', $fields ) ) {
-			$data['styles'] = $theme->get_raw_data()['styles'];
+			$raw_data       = $theme->get_raw_data();
+			$data['styles'] = isset( $raw_data['styles'] ) ? $raw_data['styles'] : array();
```

Comment:
I thought a little bit more about this and I think ideally if there are no styles we should just not include the styles in the rest response.
```
			$raw_data       = $theme->get_raw_data();
			if( isset( $raw_data['styles'] ) ) {
				$data['styles'] = $raw_data['styles'];
			}
``` 
We avoid sending unnecessary bytes over the wire and we are more consistent with what is defined in theme.json. In that case, theme.json is not defining anything for styles if theme.json defines styles: {} in that case we send the empty array.

We should just make sure the Javascript code is not crashing if styles are omitted.

Reply:
In all my testing, there was no crash if `styles` weren't set. I could use some testing though from you @jorgefilipecosta or André that you've worked so much on these parts.

Reply:
Thank you for updating your PR and testing it 👍  I will just do a final checkup and if everything seems ok commit the patch.